Multisig/Data/Services/Safe Transaction Service/SignTransactionRequest.swift

```diff
@@ -9,17 +9,18 @@
 import Foundation
 import Web3
 
+#warning("Figure out how to handle the case if private key can not be extracted from the keychain")
```

Comment:
I would rather create a new task in ZenHub for this than to have it as a warning in the source code

Reply:
Done: https://github.com/gnosis/safe-ios/issues/612